app/components/Content/Content.js

```diff
@@ -27,13 +34,40 @@ type Props = {
  * </Content>
  * ```
  */
-function Content({ banner, children, className }: Props) {
+
+function Content({ banner, youtubeParams, children, className }: Props) {
+  const [isClicked, click] = useState(false);
+  const [isLoading, setIsLoading] = useState(true);
+
   return (
-    <div>
-      {banner && (
-        <div className={cx(styles.cover, className)}>
-          <Image src={banner} />
+    <Flex column alignItems="center">
+      {!isEmpty(youtubeParams) ? (
+        <div>
+          <LoadingIndicator
+            loading={isLoading}
+            className={isLoading ? {} : styles.hidden}
```

Comment:
Hmmm.
Why not 

```jsx
<LoadingIndicator loading={isLoading}>  
          <Flex
            justifyContent={'center'}
            style={{
              maxHeight: isLoading ? '0' : isClicked ? '619px' : '358px'
            }}
            className={cx(styles.youtubeFrame, isLoading ? styles.hidden : {})}
          >
            <Youtube
              videoId={youtubeParams.v}
              opts={{ playerVars: { start: youtubeParams.t } }}
              onStateChange={() => click(true)}
              onReady={() => setIsLoading(false)}
            />
          </Flex>
</LoadingIndicator>
```

Reply:
Because the `<Youtube>` component will not load, and therefore the `onReady` function will never run. As a result, the loading indicator will not disapear 😨